tests/front-end/index.js

```diff
@@ -1,5 +1,5 @@
 const path = require('path');
-const compilerOptions = require('./tsconfig.json').compilerOptions;
+const compilerOptions = require('../../tsconfig.json').compilerOptions;
 
 process.chdir(__dirname);
 
```

Comment:
I'm not sure if the below one should also use the root `tsconfig`--struggled with the documentation: https://www.npmjs.com/package/ts-node#:~:text=ts-node%20and-,register%20the%20loader,-for%20future%20requires

Reply:
I wouldn't worry too much about it yet since the existing mocha suite runs just one example test. I'd split the work of improving or replacing the mocha harness to a new PR, and since we're reading a property out of a json file and that property is from the parent this looks right to me.

Reply:
To be absolutely correct we'd want to read both properties out of both json files and then merge together the `compilerOptions` sub-keys the way typescript does...but this is working now (it's in a better state than in `main`) and we're only missing the override of `target` and `noEmit` keys from the overridden child `compilerOptions`.

Reply:
Maybe add a tech debt card about merging the `compilerOptions` keys from the two files?

Reply:
Will combine `compilerOptions` as part of this PR